tasks.py

```diff
@@ -223,22 +215,30 @@ def docker_switching_branch(ctx):
     docker_manage(ctx, "block_inventory")
 
 
-@task
-def docker_new_db(ctx):
-    """Delete your database and create a new one with fake data"""
+def docker_stop_and_delete_db(ctx):
     print("Stopping services first")
     ctx.run("docker-compose down")
     print("Deleting database")
-    ctx.run("docker volume rm foundationmozillaorg_postgres_data")
-    print("Applying database migrations.")
-    docker_migrate(ctx)
-    print("Creating fake data")
-    docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    try:
+        ctx.run("docker volume rm foundationmozillaorg_postgres_data")
+    except:  # noqa: E722 
```

Comment:
I don't know how to get the "real" exception so that we're not doing a bare except - the log doesn't tell us in this case.

Reply:
Read through the invoke source, I think it raises one of these exceptions: https://github.com/pyinvoke/invoke/blob/aa5a02bfd7e6a73ceed11f0a94cf30d4a422b8aa/invoke/runners.py#L276-L286